api/scpca_portal/management/commands/load_data.py

```diff
@@ -80,14 +80,19 @@ def handle(self, *args, **options):
 
 
 def cleanup_output_data_dir():
-    cleanup_items = (ComputedFile.README_FILE_NAME, ComputedFile.README_SPATIAL_FILE_NAME, "*.tsv")
+    cleanup_items = (
+        ComputedFile.README_FILE_NAME,
+        ComputedFile.README_MULTIPLEXED_FILE_NAME,
+        ComputedFile.README_SPATIAL_FILE_NAME,
+    )
     for item in cleanup_items:
         for path in Path(common.OUTPUT_DATA_DIR).glob(item):
             path.unlink()
 
 
 def load_data_from_s3(
     allowed_submitters: set = ALLOWED_SUBMITTERS,
+    cleanup_output_data=not (settings.DEBUG or settings.TEST),
```

Comment:
```suggestion
    default_cleanup_output_data: bool = IS_PRODUCTION),
```

Reply:
Add to the top of the file, under `ALLOWED_SUBMITTERS`:
```python
IS_PRODUCTION = not (settings.DEBUG or settings.TEST)
```

Reply:
Also add another parameter that overrides the option `default_cleanup_output_data` so if we want to persist the data for debugging we can without changing the settings in production.

Reply:
I've added the implementation based on your comments.